core/nuxeo-operation.js

```diff
@@ -26,6 +26,7 @@ import './nuxeo-connection.js';
    *                      op="Document.Query"
    *                      params='{"query": "select * from Document"}'
    *                      on-response="handleResponse"
+   *                      on-failure="handleFailure"
```

Comment:
I guess it'd be cleaner to just have a `on-error` since this is what it's about. iirc the `unauthorized-request` thing was introduced so we could show the session timeout banner. wdyt @Gabez0r are we sending some other event that could make more sense here ?

Reply:
I agree with you @nelsonsilva, it's what I did, but I brook the tests see  [my comment](https://github.com/nuxeo/nuxeo-elements/pull/345#issue-548299867). I don't know why the fact to dispatch an `error` event break some tests. That the main the reason why I renamed it into `failure` where the best approche is `error`

Reply:
I also think we should use on-error here, even if 'error' is a rather unfortunate name, for the sake of consistency with other elements like:

- [nuxeo-page-provider](https://github.com/nuxeo/nuxeo-elements/blob/master/core/nuxeo-page-provider.js#L400-L409)
- [nuxeo-task-page-provider](https://github.com/nuxeo/nuxeo-elements/blob/master/core/nuxeo-task-page-provider.js#L239-L247)
- [nuxeo-es-search](https://github.com/nuxeo/nuxeo-elements/blob/master/dataviz/nuxeo-es-search.js#L142-L151)
- [nuxeo-document-distribution-chart](https://github.com/nuxeo/nuxeo-elements/blob/master/ui/dataviz/nuxeo-document-distribution-chart.js#L501-L505)